*********************************************************************
* LD Score Regression (LDSC)
* Version 1.0.1
* (C) 2014-2019 Brendan Bulik-Sullivan and Hilary Finucane
* Broad Institute of MIT and Harvard / MIT Department of Mathematics
* GNU General Public License v3
*********************************************************************
Call: 
./ldsc.py \
--h2 covid_ldsc.sumstats.gz \
--ref-ld-chr eur_w_ld_chr/ \
--out covid_ldsc \
--w-ld-chr eur_w_ld_chr/ 

Beginning analysis at Thu Oct 15 07:41:46 2020
Reading summary statistics from covid_ldsc.sumstats.gz ...
Read summary statistics for 2731 SNPs.
Reading reference panel LD Score from eur_w_ld_chr/[1-22] ... (ldscore_fromlist)
Traceback (most recent call last):
  File "/ldsc/ldsc.py", line 644, in <module>
    sumstats.estimate_h2(args, log)
  File "/ldsc/ldscore/sumstats.py", line 326, in estimate_h2
    args, log, args.h2)
  File "/ldsc/ldscore/sumstats.py", line 243, in _read_ld_sumstats
    ref_ld = _read_ref_ld(args, log)
  File "/ldsc/ldscore/sumstats.py", line 82, in _read_ref_ld
    'reference panel LD Score', ps.ldscore_fromlist)
  File "/ldsc/ldscore/sumstats.py", line 152, in _read_chr_split_files
    out = parsefunc(_splitp(chr_arg), _N_CHR, **kwargs)
  File "/ldsc/ldscore/parse.py", line 103, in ldscore_fromlist
    y = ldscore(fh, num)
  File "/ldsc/ldscore/parse.py", line 147, in ldscore
    first_fh = sub_chr(fh, chrs[0]) + suffix
IndexError: list index out of range

Analysis finished at Thu Oct 15 07:41:47 2020
Total time elapsed: 0.76s
